features/unsupported/USBHost/USBHostMSD/USBHostMSD.cpp

```diff
@@ -29,13 +28,16 @@
 #define GET_MAX_LUN             (0xFE)
 #define BO_MASS_STORAGE_RESET   (0xFF)
 
-USBHostMSD::USBHostMSD(const char * rootdir) : FATFileSystem(rootdir)
+USBHostMSD::USBHostMSD()
 {
     host = USBHost::getHostInst();
-    init();
+    /*  register an object in FAT */
+
+    init_usb();
 }
 
-void USBHostMSD::init() {
+void USBHostMSD::init_usb()
```

Comment:
As USBhostMSD inherits from BlockDevice and block device as a virtual method init.
It is better to rename the method relative to USB init , as init_usb . Also this method is protected so the renaming does not modify the behaviour for the user of this class.